test/issues.jl

```diff
@@ -9,6 +9,18 @@ using InteractiveUtils: @which
 include("forbidproperties.jl")
 include("simplesmatrix.jl")
 
+
+@testset "issue #15" begin
+    s = sparse([1, 2], [1, 2], [10, missing])
+    d = Matrix(s)
+    @test (sparse(d); true)
+    s2 = sparse(d)
+    @test s2[1, 1] == 10
+    @test s2[2, 1] == 0
+    @test s2[1, 2] == 0
+    @test s2[2, 2] === missing
```

Comment:
maybe also add `setindex!` tests that were failing in https://github.com/JuliaSparse/SparseArrays.jl/issues/15?